runtime.md

```diff
@@ -99,7 +97,7 @@ This operation MUST return the state of a container as specified in the [State](
 `create <container-id> <path-to-bundle>`
 
 This operation MUST [generate an error](#errors) if it is not provided a path to the bundle and the container ID to associate with the container.
-If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
+If the ID provided is not unique across all containers within the [scope of the container](#scope-of-a-container), or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
```

Comment:
@wking In my understanding, this statement means container ID should be unique within all containers created by one runtime, but two containers created by different runtimes on the same host can be the same.

Your change to `unique id across all containers within the scope of the container` doesn't make sense to me. I think it's not `id within the scope of xxx`, but `all containers within the scope of xxx`.